app/experimenter/templates/experiments/edit_objectives.html

```diff
@@ -25,3 +41,8 @@ <h4><span class="fas fa-info-circle"></span> Objectives &amp; Analysis</h4>
     Describe both in detail here.
   </p>
 {% endblock %}
+
+{% block extrascripts %}
+    <script src="{% static "js/edit-objectives.js" %}"></script>
+
```

Comment:
extra newline?

Reply:
i removed some of all the extra new lines this file got 🤦‍♀